plugins/EnviroSense.Plugins.PostgresRepositories/Migrations/Scripts/SQL/V10__Alter_table_device_api_keys.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE device_api_keys
+RENAME TO api_keys;
+ALTER TABLE api_keys DROP CONSTRAINT device_api_keys_device_id_fkey;
+ALTER TABLE api_keys RENAME COLUMN device_id TO account_id;
```

Comment:
This won't work, because `device_id` is different from `account_id`. Migration will fail with existing data.

What we should do instead:
- create new column, nullable
- fill created column with account id (every row has `device_id`, so we can then, determine `account_id`, by looking in devices table)
- make column not null, add foreign key
- drop existing `device_id` column